### ISSUE TITLE
Some Markdown format seems to be breaking Confluence

### ISSUE THREAD

**Issue by carlosjgp** [szn/confluence.md#10]
I'm seeing the same error as https://github.com/szn/confluence.md/issues/8

Markdown2 dependency has at least 2 new releases since the last `confluence.md` release with bug fix
https://github.com/trentm/python-markdown2/blob/master/CHANGES.md

Maybe this could be fix by simply updating this dependency

**Comment by carlosjgp** [comment]
Looks like these are code blocks or even "code" format

```
` ` `
code block
` ` `
```

or inline code like 
```
my phrase with `code`
```


https://community.atlassian.com/t5/Confluence-questions/Code-Macro-via-Confluence-REST-API/qaq-p/2097123

**Comment by carlosjgp** [comment]
Nope, I was wrong. The problem is a link with URL parameters on it

```
You would need to get access to [`PagerDuty` Application group](https://access-control.my.domain/access-request/new/?application_group=PagerDuty&permanent_access=true)
```

By changing this to 
```
You would need to get access to `PagerDuty` Application group
- https://access-control.my.domain/access-request/new/?application_group=PagerDuty&permanent_access=true
```

the update goes through

**Comment by szn** [comment]
@carlosjgp I upgraded dependencies (see v.0.2.5). This didn't help. The problem is that for some reason `atlassian` python package has a problem with URL's with two parameters. The shortest example is:

```md
[URL](https://sample.url/?a=1&b=2)
```

This is working:

```md
[URL](https://sample.url/?a=1)
```

I am trying to debug the problem in the `atlassian-python-api` repo.

**Comment by carlosjgp** [comment]
Thank!
